### ISSUE TITLE
Crashing when initializing with latest version MekanismGenerators-1.12.2-9.4.2.327.jar 

### ISSUE THREAD

**Issue by Unrepentant-Atheist** [aidancbrady/Mekanism#4827]
-----------------------------Dev note-------------------------------
This issue has been fixed and will be available in build 329. Please do not repost.
For personal use in the meantime (only Aidan is able to make Curse releases), you may use this build: 
http://maven.thiakil.com/mekanism/Mekanism/1.12.2-9.4.3.329-thiakil/Mekanism-1.12.2-9.4.3.329-thiakil.jar
-----------------------------------------------------------------------

https://hastebin.com/pulohireto.sql

**Comment by JasonKillsUs** [comment]
same here, exactly the same crash_log MC 1.12.2_2554 mk 327

**Comment by PWaksman** [comment]
Just to add that it happens also in 328.

**Comment by thiakil** [comment]
What do you mean 328?

**Comment by PWaksman** [comment]
The version 9.4.2.328 for 1.12.2

**Comment by thiakil** [comment]
That's strange, that shouldn't have built from a commit on the 1.12 branch.

And the commit completely removes the line that's crashing, so it is impossible to happen with my commit. Please ensure you updated the main mod and not just the Generators jar. (You should *never* have mismatched Mekanism jars)

**Comment by thiakil** [comment]
Ok I have been informed 328 is the same as 327

**Comment by PWaksman** [comment]
no problem, I was just pointing that out, as the title is for 327.

**Comment by vrork** [comment]
Confirming, happens with me too.

**Comment by EncryptedData** [comment]
Can confirm is crashing on 1.12.2-9.4.2.327 and 1.12.2-9.4.2.328.

**Comment by thiakil** [comment]
@vrork @EncryptedData please don't ever post a "me too" to an issue unless the dev has said they can't reproduce it. Especially one that has been marked as fixed in dev.

**Comment by MasterBuilder747** [comment]
@thiakil why tf did you close this issue? v327 IS the same as v328, and after waiting for 35 minutes (yes that's right!) for my pack to load, i still get the crash after updating! Please reopen this issue now! This NEEDS to be fixed asap!

**Comment by thiakil** [comment]
328 is a rebuild of 327, it was a server test. As for closing it, that was github when I merged stuff to master. Calm your tits.

**Comment by TimGoll** [comment]
Any news about this bug being fixed?

**Comment by thommy101** [comment]
The bug is fixed, it's just waiting on a new release.. @thiakil and I can't make a release ourselves (yet). It currently waits on an action from @aidancbrady

**Comment by TimGoll** [comment]
Okay, sounds cool. :)

**Comment by thiakil** [comment]
If your desperate, you can use this build http://maven.thiakil.com/Mekanism/1.12.2-9.4.3.329-thiakil/

**Comment by TimGoll** [comment]
Solved it now like this, until you publish an official release (red text)
https://minecraft.curseforge.com/projects/engineers-forest-3

**Comment by InterPlay02** [comment]
Cool!!!

I was about to post a new issue with this same crash.
https://pastebin.com/8Guhmz8J

But, if is everything fixed, ok....
I can wait until you guys release the new version.
(Cause I can't figure out what I should download in that link @thiakil posted.)

Thank you guys, for the awesome mod!

**Comment by TimGoll** [comment]
Just download `Mekanism-1.12.2-9.4.3.329-thiakil.jar`

**Comment by PWaksman** [comment]
Where?

**Comment by kreezxil** [comment]
More confirmation.

Issue:  9.4.2 crashes dedicated server Forge 1.12.2 build 2566

Notes:
     complains about gasses not being registered before postInit

crash report:
https://gist.github.com/35227b08b7a3aff77fe890591b4b3487

fml server latest:
https://gist.github.com/e01ede766a83c96c81c8852fdf4f31d6

**Comment by thiakil** [comment]
@kreezxil Don't do that for an issue *already marked as fixed in dev*, it is 100% unnecessary.

**Comment by thiakil** [comment]
330 has been released
